allows us to build the standalone scudo via a cmake flag instead of by hand or
needing all the sanitizers.

the additional not guard prevents double-inclusion if regular sanitizers are enabled
diff --git a/compiler-rt/lib/CMakeLists.txt b/compiler-rt/lib/CMakeLists.txt
index 18eed24..cd8e901 100644
--- a/compiler-rt/lib/CMakeLists.txt
+++ b/compiler-rt/lib/CMakeLists.txt
@@ -70,6 +70,10 @@ if(COMPILER_RT_BUILD_ORC)
   compiler_rt_build_runtime(orc)
 endif()
 
+if(COMPILER_RT_BUILD_SCUDO_STANDALONE AND NOT COMPILER_RT_BUILD_SANITIZERS)
+  add_subdirectory(scudo/standalone)
+endif()
+
 # It doesn't normally make sense to build runtimes when a sanitizer is enabled,
 # so we don't add_subdirectory the runtimes in that case. However, the opposite
 # is true for fuzzers that exercise parts of the runtime. So we add the fuzzer
